Northstar.Client/mod/scripts/vscripts/ui/menu_mod_settings.nut

```diff
@@ -641,6 +708,19 @@ void function CustomButtonPressed( var button )
 	c.onPress()
 }
 
+void function ColorButtonPressed( var button )
+{
+	var panel = Hud_GetParent( button )
+	ConVarData c = file.filteredList[ int( Hud_GetScriptID( panel ) ) + file.scrollOffset ]
+	// c.onPress()
+	// printt(c.displayName, c.type, c.conVar)
+
+	SetConVarString("ModSettings.current_color_convar", c.conVar)
```

Comment:
You forgot to rename it, and other instances of this convar to ``modemenu_current_color_convar``

Reply:
oh...thank you